### ISSUE TITLE
Add back option for hiding watch and fork count

### ISSUE THREAD

**Issue by jhpratt** [refined-github/refined-github#5301]
### Description

Please re-add the option that hides the watch and fork count. I want to see them on every page, not just some. Actively removing options does not help usability.

### Example URL

https://github.com/refined-github/refined-github

**Comment by kidonng** [comment]
You can easily revert the style: https://github.com/refined-github/refined-github/pull/5197#issuecomment-993390710

We would like to see more opinions on this to decide whether we should make it an option.

> Actively removing options does not help usability.

Please read https://github.com/refined-github/refined-github#customization.

**Comment by jhpratt** [comment]
I'm aware it's an opinionated project. The only thing I meant by that statement is that options that already exist shouldn't be removed in favor of enabling it all the time. Surely there was a reason for having it be an option in the first place?

**Comment by fregante** [comment]
The option exists, via CSS:

```css
.pagehead-actions :is(#repo-network-counter, #repo-notifications-counter) {
	display: inline-block;
}
```

**Comment by jhpratt** [comment]
I will ask this: why was the option removed? As I said last week

> Surely there was a reason for having it be an option in the first place?

There are _hundreds_ of options. It's not like removing this one keeps it lean.

**Comment by fregante** [comment]
> Actively removing options does not help usability.

It does, because it saves space

## Before

<img width="828" alt="Screen Shot 6" src="https://user-images.githubusercontent.com/1402241/150668975-7ac1203a-b3b7-4990-865f-bd727e0c962f.png">


## After

<img width="800" alt="Screen Shot 7" src="https://user-images.githubusercontent.com/1402241/150668980-bd140969-93fd-45ce-b73e-88c22e7ae1a8.png">

**Comment by jhpratt** [comment]
Err...that's not at all what I meant. That's a reason for it to _exist_; it's not a reason to remove the option that toggles it. My browser is always full width; I have never experienced wrapping like that (nor will I).

**Comment by jhpratt** [comment]
I use the refined GitHub extension for only a select few features, having an overwhelming majority of features disabled. I don't believe I should have to use custom CSS to _undo_ something that could previously be done with a simple checkbox. If I wanted to do that, I'd just have user CSS set; I have this extension because it's supposed to make things _easier_ — something that is not currently the case.

I understand this is an opinionated project, but given that there are over 200+ options, I don't see the reason in removing one. I've disabled this extension until this is added back, and will uninstall it if it's not. I'm sure you won't miss a single user, but this is a major change to me with no clear reason. _That_ is the problem.

**Comment by kidonng** [comment]
> I don't believe I should have to use custom CSS to _undo_ something that could previously be done with a simple checkbox.

It used to a standalone CSS feature because the way we hide it is not obvious to everyone and requires a click to reveal the information.

But that's not the case anymore, now we simply remove the duplicate info that can already be seen on the repository homepage.

Note GitHub moved the stargazer, watcher and fork links to the homepage so people **have to** go there anyway.

> If I wanted to do that, I'd just have user CSS set

It's already stated on the settings page, the custom CSS _is_ for you to revert styles you don't want.

> I have this extension because it's supposed to make things _easier_ — something that is not currently the case.

How? We already provided the solution and all you need to do is to copy and paste the overriding styles. It is even synced with your browser.

If you mean it's not easier because you need to open an issue to ask how to disable the styles, I'm sorry and we are aware that better docs are needed.

> I understand this is an opinionated project, but given that there are over 200+ options, I don't see the reason in removing one.

We load CSS-only feature all the time as it makes things faster for everyone, and the few people who want to disable it can use custom CSS.

As stated above, we believe the feature at it's current state does not meet the requirement to be standalone (controversial, not obvious etc.).

> I've disabled this extension until this is added back, and will uninstall it if it's not. I'm sure you won't miss a single user, but this is a major change to me with no clear reason. _That_ is the problem.

We can't satisfy everyone, not to mention this is an opinionated project.

Here is a steak, you care about how it's done even if it tastes the same. The problem is most people don't. If enough people are concerned (doesn't have to be many, usually 3 or 4 would do) we would certainly reconsider. 

In short, we won't make any changes for the moment, hope you can understand. You decide what you want to do.

**Comment by fregante** [comment]
Since #6333 (version 23.2.11) you can disable the feature in the options page.
It's now called `clean-repo-header`
